templates/redmine/0/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '2'
+
+services:
+  redmine-datavolume:
+    image: "busybox"
+    volumes:
+      - /home/redmine/data
+      - /var/lib/mysql
```

Comment:
on all of this volume stuff, how about I ditch the busybox container and use the version 2 volumes: block?  Pushing an example next